### DISCUSSION TITLE
2022-01-19 Meeting

### DISCUSSION THREAD

**Discussion by lorenzolewis** [lorenzolewis/Elements#5] [Meeting Notes]
I'm some sample meeting notes

You can update me as we move along

**Comment by lorenzolewis** [comment]
Oooo, I'll contribute here!

**Comment by yankeeinlondon** [comment]
## Tentative Agenda:

- Introduce idea of "guiding principles"
    - high-level structure that is used to simplify decisions for team and provide clarity to outside contributors by reinforcing consistency
- Review Docs and Search POV through guiding principles
   > A list of guiding principles can be much further reaching that Docs and Search but we'll use this as an example to achieve the goal of 
    - we'll present what we think should be the desired "end state" worded in firm language as if this were the decision and then we'll ask the group how they'd want to change it
-  Go into the details for "day 1"
    - [ ] Lorenzo presents Docs
    - [ ] Ken presents Search
- Next steps
    - [ ] I think we should take the next step of updating all content based on discussion in meeting
   -  [ ] We should try to agree where the "Guiding Principles" should live but I'm hoping we can agree on Docs repo

**Comment by yankeeinlondon** [comment]
## `tauri` Repo ownership

![image](https://user-images.githubusercontent.com/980757/150199815-77f3e69f-3424-4ab0-969f-2e3dfa98532f.png)

**Reply by yankeeinlondon** [reply]
I'll get this into a repo so you can directly contribute to markdown soon

**Comment by yankeeinlondon** [comment]
## `tauri-docs` Repo Ownership

![image](https://user-images.githubusercontent.com/980757/150206168-1bfccac2-3196-41f0-a976-19d821ea999e.png)

![image](https://user-images.githubusercontent.com/980757/150201090-ff88a5bc-2319-4b8c-ae2b-8c57f21e1062.png)
